crawler/mesos.py

```diff
@@ -0,0 +1,237 @@
+#! /usr/bin/python
```

Comment:
I thought about that. The code will be useful in the future even though I don't use it now. But, if it lowers the coverage, I can remove